### DISCUSSION TITLE
Amd Ryzen 3200u, vega3 stuck

### DISCUSSION THREAD

**Discussion by FlyingThaCat** [ChefKissInc/NootedRed#22] [General]
Im already use the latest lilu and I download from the latest CI and when i use it, it stuck on this progress what should I do
![20230308_120624](https://user-images.githubusercontent.com/95353217/223625541-698be45b-1fb3-4a64-9100-7bd1ad6fba46.jpg)

**Comment by VisualEhrmanntraut** [comment]
These issues aren't for requesting support. Use of experimental software means you will have issues. Some common user-side configuration problems are: Not using MacBookPro16,3 or MacPro7,1 SMBIOS, using WhateverGreen, not using Lilu 1.6.4 or later (Official release was made last Monday). Some issues we don't know how to resolve yet are: random AGDP-related black screens, and sometimes, but very rarely, random early panics

**Reply by FlyingThaCat** [reply]
Im already use MacBookPro16,3 smbios, i dont have whatever green enabled in config.plist, i just change the lalu to version 1.6.4 maybe i will try to install first and apply the patch when the os is finally installed

**Comment by VisualEhrmanntraut** [comment]
Oh, and, recoveryOS might get stuck in verbose using this kext (including updates), there is an attempt made to fix it, but its effectiveness is not apparent yet

**Comment by FlyingThaCat** [comment]
I already try it and you are correct when it is in recoveryOS it just stuck on verbose using it after installed works like a charm

**Reply by VisualEhrmanntraut** [reply]
Great! Glad to know it works. Enjoy. Hopefully we fully fix the recoveryOS part
